package.json

```diff
@@ -29,7 +29,8 @@
     "deploy-deployer": "npx hardhat run scripts/core/deployContractFactory.ts --network",
     "/////// deploy-test ///////": "echo 'deploy test scripts'",
     "create-profile": "npx hardhat run scripts/test/createProfile.ts --network",
-    "create-pool": "npx hardhat run scripts/test/createPool.ts --network"
+    "create-pool": "npx hardhat run scripts/test/createPool.ts --network",
+    "smock": "smock-foundry --contracts contracts/core"
```

Comment:
Should we specify the full path? we currently only want Allo.sol to get generated

Reply:
I tried but fails with `[Error: ENOTDIR: not a directory, scandir '/.../.../.../.../contracts/core/Allo.sol']`. I think smock only allows folders